lib/config/presets/internal/group.ts

```diff
@@ -444,6 +445,13 @@ const staticGroups = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  remark: {
+    description: 'Group remark packages together.',
+    matchSourceUrlPrefixes: [
+      'https://github.com/remarkjs/',
+    ],
+    matchPackagePrefixes: ['remark-'],
```

Comment:
you are aware that both matchers are combined with `and`? so package `remark` is not matched with that rule.

should we also limit the datasource?

Reply:
The reason I had both was this (being explicit about first-party data sources, while still allowing 3rd party packages):

- https://github.com/renovatebot/renovate/pull/25969#discussion_r1405655906

Reply:
But I get the feeling that there's resistance here with doing that so I removed the 3rd-party packages

- https://github.com/renovatebot/renovate/pull/25969#discussion_r1410311514

Reply:
I'm not sure what you mean by limiting the data source - maybe you mean limiting matched packages to only come from npm?

Reply:
Yes, that what he meant like `matchDatasources: ['npm']` 

Does remark publishes other kinds of artifacts? e.g. Docker images?